source/fuzz/fuzzer_util.h

```diff
@@ -371,7 +372,7 @@ uint32_t MaybeGetScalarConstant(
     opt::IRContext* ir_context,
     const TransformationContext& transformation_context,
     const std::vector<uint32_t>& words, uint32_t scalar_type_id,
-    bool is_irrelevant);
+    bool is_irrelevant = false);
```

Comment:
There is still a default parameter here.

Reply:
Fixed